src/ConfigurationProcessor.php

```diff
@@ -62,21 +75,41 @@ public function processInstallation(array $configuration)
         }
 
         $nativeModules = $configuration['modules'];
-        $processManager = new ProcessManager(100, 8);
+        $timeout = array_key_exists('timeout', $configuration) ?
```

Comment:
isset is faster than array_key_exists (and i'm sure you don't care about null values).

Reply:
I don't care either about this kind of micro-optimizations, but I do care about a piece of valuable information: $configuration is an array :+1:

Reply:
long story short: if $configuration is not an array this code will throw a warning, converted into exception by PrestaShop

if I use an isset, this will return `false` (and composer install won't install any modules)

Reply:
`$configuration` is known to be an array because of parameter typing. Just change it to `isset` already 😉